### ISSUE TITLE
New Refinery CMS website

### ISSUE THREAD

**Issue by bricesanchez** [resolve/refinerycms-website#4]
Hi @djones and @parndt,

Could we use this repo for the new refinery website:
https://github.com/refinery/website

**Comment by bricesanchez** [comment]
Done :)

**Comment by djones** [comment]
Hi @bricesanchez,

This is much more mobile friendly which is nice! 👍 I think the changes here are mostly good but there are a number of regressions. I've created individual issues for each of them on the new repo.

**Comment by djones** [comment]
Upon further review... the formatting on the guides is very broken. Seems like a major regression there that will make them very hard to follow.

**Comment by parndt** [comment]
Yeah, the docrails stuff that was used before was fairly old and hard too use. Keen for any help you can give on making them nicer, and as a bonus they're now backed by markdown, not textile (finally!). https://github.com/refinery/website/blob/master/app/workers/guides_worker.rb
